lua/core/compe.lua

```diff
@@ -92,8 +100,9 @@ M.setup = function()
       return t "<Plug>(vsnip-jump-next)"
     elseif check_back_space() then
       return t "<Tab>"
+    elseif is_emmet_active() then
```

Comment:
As discussed, please add `not check_back_space() and` condition before is_emmet_active() to allow use of <tab> at the beginning of the line before non-space chars have been inserted.

Reply:
I think moving check_backspace before is_emmet_active will have the same effect and avoid calling check_back_space twice.  If there's a space, it will return a tab and never reaches the emmet check.